.babelrc

```diff
@@ -1,4 +1,7 @@
 {
-  "plugins": ["transform-class-constructor-call"],
-  "presets": ["es2015", "stage-2"]
+  "presets": ["es2015", "stage-2"],
```

Comment:
I recently added what "feels" like a minimal Babel/Jest/Yarn setup in our Auth0 sync code. Might be useful to standardize on.

Reply:
I'll give it a shot. Might have to wait for my follow up ticket for upgrading the js styling here. I'm not sure what `stage-2` actually means for example....

Reply:
The `stage-2` plugin is the only difference between your set up and what's here. `install` breaks with out it. We can come back to this in: https://globality.atlassian.net/browse/GLOB-18988